src/quo2/components/icons/svg.cljs

```diff
@@ -0,0 +1,48 @@
+(ns quo2.components.icons.svg
+  "Declare icons in this namespace when they have two possible colors, because the
+  ReactNative `:tint-color` prop affects all non-transparent pixels of PNGs. If
+  the icon has only one color, prefer a PNG.
+
+  Keep all SVG components private and expose them by name in the `icons` var."
+  (:require [react-native.svg :as svg]
+            [quo2.foundations.colors :as colors]))
+
+(defn- container
+  [{:keys [size accessibility-label style]
+    :or   {size 20}}
+   & children]
+  (into [svg/svg
+         {:accessibility-label accessibility-label
+          :style               style
+          :width               size
+          :height              size
+          :view-box            (str "0 0 " size " " size)
+          :fill                :none}]
+        children))
+
+(defn- clear-20
```

Comment:
Why do we need to use hardcoded size here? Is it better to have a size options like in the `container` above?

Reply:
@ajayesivan, the SVG exported by Figma only works correctly with a specific dimension, 20 in this case. The SVG generated by Figma for size 12 is different and I checked other icons and it's the same story. So I chose to be as explicit as possible and name the var with the size. The only thing that was safe to generalize was the SVG viewbox, which I extracted into the `container` component.

Naming the var with the size was the way I found to tell other devs that this icon var should not be used for other sizes.

It is in fact possible to generalize the icons to different sizes by manipulating the `path.d`  prop, but I felt this was out of scope for this issue.

Reply:
It's also good that all of these vars are private, and we can safely improve them in the future without causing breakage.

Reply:
Ideally, we should be able to use a single SVG for all size variations, and I think thats one the major advantages of using SVG over PNG. Maybe we can discuss this with design team and come up with a better solution.

Reply:
Agree with @ajayesivan, I think that the point of using vectorized icons is to use one resource for all sizes without loosing quality. Ideally we should export it once.

Also wondering if we add lots of icons how big this file will end up being.

Reply:
Totally agree with you guys. If we want to replace PNGs with SVGs in the future, we'll need to coordinate with designers a bit of work, especially to optimize them for reusability for different sizes. The exported SVGs from Figma are fine, but not ideal.